src/renderer/components/playlist-info/playlist-info.vue

```diff
@@ -22,10 +22,9 @@
         {{ title }}
       </h2>
       <p>
-        {{ videoCount }} {{ $t("Playlist.Videos") }} - <span v-if="!hideViews">{{ viewCount }} {{ $t("Playlist.Views") }} -</span>
-        <span v-if="infoSource !== 'local'">
-          {{ $t("Playlist.Last Updated On") }}
-        </span>
+        {{ videoCount }} {{ $t("Playlist.Videos") }} <span v-if="!hideViews && infoSource !== 'piped'">- {{ viewCount }} {{ $t("Playlist.Views") }} </span>
```

Comment:
View count missing in Local API and that should be hidden
https://youtube.com/playlist?list=PL8mG-RkN2uTzaEZfV9CUBOQtAtcnWSTuJ
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/1cbd4090-f507-4cc6-a2ba-65d0274be1b8)

`hideViews` is not needed due to it's used already used to set `viewCount` in `mounted`
```suggestion
        {{ videoCount }} {{ $t("Playlist.Videos") }} <span v-if="viewCount != null">- {{ viewCount }} {{ $t("Playlist.Views") }} </span>
```

Reply:
Hmm, weird. This is what I see when using local api 
![image](https://github.com/FreeTubeApp/FreeTube/assets/78101139/d93598a4-af9a-4973-ae6b-3bb930bd2672)

Reply:
Yup I see 0 views now
But it doesn't matter, the code has no need to know the backend, only the presence of data